Formula/uvg266.rb

```diff
@@ -0,0 +1,32 @@
+class Uvg266 < Formula
+  desc "Open-source VVC/H.266 encoder"
+  homepage "https://github.com/ultravideo/uvg266"
+  url "https://github.com/ultravideo/uvg266/archive/refs/tags/v0.4.1.tar.gz"
+  sha256 "9d4decb1b9141ce7a439710a747db7ef0983fa647255972294879122642b8f2b"
+  license "BSD-3-Clause"
+  head "https://github.com/ultravideo/uvg266.git", branch: "master"
+
+  depends_on "cmake" => :build
+
+  resource "videosample" do
+    url "https://samples.mplayerhq.hu/V-codecs/lm20.avi"
+    sha256 "a0ab512c66d276fd3932aacdd6073f9734c7e246c8747c48bf5d9dd34ac8b392"
+  end
+
+  def install
+    # Avoid uvg266 referencing libuvg266.dylib as @rpath/libuvg266.dylib
```

Comment:
Any reason you want to avoid this? We may not have to with `CMAKE_INSTALL_RPATH` (with the benefit of very marginally faster bottle pour times).

Reply:
Without this, the test fails on Apple Silicon:
```
==> /opt/homebrew/Cellar/uvg266/0.4.1/bin/uvg266 -i lm20.avi --input-res 16x16 -o lm20.vvc
  dyld[21185]: Library not loaded: '@rpath/libuvg266.dylib'
    Referenced from: '/opt/homebrew/Cellar/uvg266/0.4.1/bin/uvg266'
    Reason: tried: 'lib/libuvg266.dylib' (no such file), './libuvg266.dylib' (no such file), '../lib/libuvg266.dylib' (no such file), 'lib/libuvg266.dylib' (no such file), './libuvg266.dylib' (no such file), '../lib/libuvg266.dylib' (no such file), '/usr/local/lib/libuvg266.dylib' (no such file), '/usr/lib/libuvg266.dylib' (no such file)
```

I have tried `-DCMAKE_INSTALL_RPATH=#{rpath}` alone but the resulting reference is to `@rpath/libuvg266.dylib`, which seems that is the underlying issue. Let me know if I missed something or if there's a more idiomatic way to avoid this.

Note that upstream has some config related to rpath in their `CMakeLists.txt`, see https://github.com/ultravideo/uvg266/blob/master/CMakeLists.txt#L130-L133. I've tried commenting these out but the resulting rpath behavior appears to be the same.